packages/css/src/components/button/button.template.ts

```diff
@@ -11,17 +11,20 @@ export function buttonTemplate(button: Button | ButtonAnchor) {
 }
 
 function anchorElement({
+  variant,
   url,
   label,
   modifier,
   id,
   icon,
   iconMode
 }: ButtonAnchor) {
+  const className = variant && modifier ? `dso-${variant}` + ' ' + modifier : variant ? `dso-${variant}` : modifier ? modifier : '';
```

Comment:
1. Dit kan leesbaarder;
2. Dubbele logica in de buttonTemplate();
3. Als er geen `!variant && !modifier` dan wordt `className = '';`, waarmee er er dus een leeg `class` attribuut komt. `ifDefined()` haalt de class alleen weg als de gegeven waarde `undefined` is.

Verplaats deze oneline naar een leesbare routine in een functie. Roep deze functie in beide template functions aan. Zorg ervoor dat er geen leeg class attribuut komt.

Reply:
Ik pas het aan